.github/workflows/kata-deploy-push.yaml

```diff
@@ -9,7 +9,8 @@ on:
       - synchronize
       - labeled
       - unlabeled
-  push:
+    paths:
+      - tools/**
```

Comment:
@egernst, I'd prefer having this is a different PR, so we can have more discussions on whether or not to keep this action.

Reply:
Does kata-deploy read values from `versions.yaml`? If yes, then I think that file should be included in the list of paths.

Reply:
The build scripts do reference -- good suggestion.